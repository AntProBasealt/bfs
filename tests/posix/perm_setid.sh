bfs_diff rainbow -perm -u+s -o -perm -g+s
