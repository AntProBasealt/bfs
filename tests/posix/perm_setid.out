rainbow/sgid
rainbow/sugid
rainbow/suid
